qiskit/transpiler/exceptions.py

```diff
@@ -49,3 +49,7 @@ def __init__(self, *msg):
     def __str__(self):
         """Return the message."""
         return repr(self.msg)
+
+
+class CircuitToWideForTarget(TranspilerError):
```

Comment:
Missing an "o" in "too"

Reply:
Lol, of course I made a typo here...

Reply:
Fixed in: https://github.com/Qiskit/qiskit/pull/11241/commits/5c90e25f4ac21f2f0028148d08f710faa6d6ae84